test/execution/compiler_test.cpp

```diff
@@ -873,6 +874,250 @@ TEST_F(CompilerTest, SimpleIndexNestedLoopJoinMultiColumnTest) {
   checker.CheckCorrectness();
 }
 
+// NOLINTNEXTLINE
+TEST_F(CompilerTest, SimpleInsertTest) {
+  // SELECT COUNT(*) FROM test_1 WHERE test_1.colB == 23
+  // INSERT INTO test_1 (colA, colB, colC, colD) VALUES (0,23,2,3)
```

Comment:
Instead of inserting into `test_1`, maybe insert into the empty table? That way you can assume that the initial size is 0 and reduce the complexity of the test.
Even with `test_1`, you can assume an initial size of `TEST1_SIZE`